src/test/java/cz/cvut/kbss/termit/persistence/dao/TermAssignmentDaoTest.java

```diff
@@ -254,8 +254,8 @@ void getAssignmentsInfoByResourceRetrievesInformationAboutAssignmentsAndOccurren
             assertEquals(term.getPrimaryLabel(), rta.getTermLabel());
         });
         final Optional<ResourceTermAssignments> occ = result.stream()
-                                                            .filter(rta -> rta instanceof ResourceTermOccurrences)
-                                                            .findAny();
+                .filter(rta -> rta instanceof ResourceTermOccurrences)
```

Comment:
why not `ResourceTermOccurrences.class::isInstance` ?

Reply:
What difference does it make?

Reply:
Not much, just a variable-free syntax only. 
Feel free to ignore.